components/cta_bar/index.styl

```diff
@@ -1,14 +1,69 @@
 @import '../stylus_lib'
 
+.cta-bar.editorial-signup
+  padding 0
+  background none transparent
+  background linear-gradient(to bottom, rgba(0,0,0,0), rgba(0,0,0,.25))
+  display flex
+  align-items flex-end
+  min-height 35vh
+  .vam-outer
+    background-color white
+    color black
+    padding 10px 20px
+    margin 10px
+    width 100%
+    border 1px solid #E5E5E5
```

Comment:
we should use the colors from the style guide here as well! `gray-lighter-color`